braindecode/models/sleep_stager_chambon_2018.py

```diff
@@ -133,10 +133,8 @@ def __init__(
             )
 
     def _len_last_layer(self, n_channels, input_size):
-        self.feature_extractor.eval()
```

Comment:
why do you say it's redundant? layers like drop out need it and it's not the same as no_grad

Reply:
The method operates on output shapes, which are unaffected.